webrtc-stats.html

```diff
@@ -1558,6 +1559,15 @@
                 </p>
               </dd>
               <dt>
+                <dfn><code>turnUrlProtocol</code></dfn> of type <span class=
+                "idlMemberType"><a>DOMString</a></span>
+              </dt>
+              <dd>
+                <p>
+                  It is the protocol used by the endpoint to communicate with the TURN server, this is only present for local candidates. Valid values for the TURN URL protocol is one of <code>udp</code>, <code>tcp</code>, or <code>tls</code>.
```

Comment:
> ", this is only present for..."

This is a separate sentence fragment; should separate with a semicolon or period.

Would also be good to break this into multiple lines.